app.js

```diff
@@ -1,8 +1,78 @@
 // import functions and grab DOM elements
+import { makeStatsString } from './utils.js';
+const skylineDropdown = document.querySelector('#skyline-dropdown');
+const waterfrontDropdown = document.querySelector('#waterfront-dropdown');
+const castleDropdown = document.querySelector('#castle-dropdown');
 
-// let state
+const skylineImageEl = document.querySelector('#skyline-image');
+const waterfrontImageEl = document.querySelector('#waterfront-image');
+const castleImageEl = document.querySelector('#castle-image');
+
+const sloganInputEl = document.querySelector('#slogan-input');
+const sloganButton = document.querySelector('#slogan-button');
+const sloganListEl = document.querySelector('.list');
+const countEl = document.querySelector('.count'); 
 
+// let state
+let skylineCount = 0;
+let waterfrontCount = 0;
+let castleCount = 0;
+// set state for all of the slogan inputs
+let sloganInput = [];
 // set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+skylineDropdown.addEventListener('change', () => {
+    // get the value from the dropdown element
+    const id = skylineDropdown.value;
+    // use that value to change the correct image tag's `.src` property
+    skylineImageEl.src = `./assets/skyline-${id}.jpeg`;
+    //count of that dropdown will increment
+    skylineCount++;
+    // update the DOM to reflect new count by calling a function
+    displayStats();
+});
+
+waterfrontDropdown.addEventListener('change', () => {
+    const id = waterfrontDropdown.value;
+    waterfrontImageEl.src = `./assets/waterfront-${id}.jpeg`;
+    waterfrontCount++;
+    displayStats();
+});
+
+castleDropdown.addEventListener('change', () => {
+    const id = castleDropdown.value;
+    castleImageEl.src = `./assets/castle-${id}.jpeg`;
+    castleCount++;
+    displayStats();
+});
+
+sloganButton.addEventListener('click', () => {
+    // get the value of the slogan input
+    const newInput = sloganInputEl.value;
+    // push the new slogan to the sloganInput array in state
+    sloganInput.push(newInput);
+    // clear out the form input's value so it's empty to the user
+    sloganInputEl.value = '';
+    // update the dom to show the new slogan by calling a function
+    displaySlogans();
+
```

Comment:
You are clearing the input `sloganInputEl.value = ''` in your event listener and your `displaySlogans` function.  Is it necessary to do this in both places?